### PR TITLE
AOM-64: List installed modules alongside installed OWAs

### PR DESCRIPTION
**Pull request by mkiterian**
## JIRA TICKET NAME:
[AOM-64 List installed modules alongside installed OWAs](https://issues.openmrs.org/browse/AOM-64)

### SUMMARY:
This PR enables installed modules to be listed with the installed OWAs

### CODE REVIEW

**Comment by Farhanisto** [comment]
Good job

File: app/js/components/manageApps/ManageApps.jsx [hunk 1]
```diff
@@ -73,34 +73,52 @@ export default class ManageApps extends React.Component {
...
         msgType: "warning",
         showMsg: true,
       });
-    }    
+    }
   }
 
   handleApplist() {
-    const resultData = [];
+    let installedOwas = [];
```
Comment (tduyile04) [inline app/js/components/manageApps/ManageApps.jsx]: This should be `const` not `let`. const prevents re-assignment not mutation. You are only mutating the array and not re-assigning, hence, const is preferred for this scenario.

**Review by tduyile04 — COMMENTED** [review]
Looks good. Great job

File: app/js/components/manageApps/ManageApps.jsx (hunk 1 above)
Comment (joel-ace) [reply app/js/components/manageApps/ManageApps.jsx]: @tolupatrick004 nice observation.

Comment (mkiterian) [reply app/js/components/manageApps/ManageApps.jsx]: 👍

**Pull request merged**